.github/workflows/ci.yml

```
@@ -0,0 +1,64 @@
+jobs:
+  test:
+    defaults:
+      run:
+        shell: bash -el {0}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python: ['3.8', '3.9', '3.10', '3.11']
```

```suggestion
        python: ['3.9', '3.10', '3.11']
```

I'd drop Py3.8, it's not supported by NumPy (see [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html)).

---

Done!